File: Android App/app/src/main/java/com/makuhita/StockPredAI/entities/Price.kt
package com.makuhita.StockPredAI.entities

class Price {
    var open: Double = 0.0
    var close: Double = 0.0
    var high: Double = 0.0
    var low: Double = 0.0
    var volume: Int = 0
    override fun toString(): String {
        return "Price(open=$open, close=$close, high=$high, low=$low, volume=$volume)"
    }
}

File: Python Backend/entities/Price.py
class Price:
    def __init__(self, open, close, high, low, volume):
        self.open = open
        self.close = close
        self.high = high
        self.low = low
        self.volume = volume

    def to_dict(self):
        return {'open': self.open, 'close': self.close, 'high': self.high,
                'low': self.low, 'volume': self.volume}
